src/plans/details/tabs/Details/components/SettingsSection/components/TargetNodeSelector/TargetNodeSelectorHelperText.tsx

```diff
@@ -0,0 +1,19 @@
+import { ForkliftTrans } from 'src/utils/i18n';
+
+import { t } from '@utils/i18n';
+
+export const TARGET_NODE_SELECTOR_DETAILS_ITEM_DESCRIPTION = t(
+  'Specify node labels that will be applied after migration to all target virtual machines of the migration plan for constraining virtual machines scheduling to specific nodes, based on node labels. This will ensure that the migrated virtual machines will run on nodes with required capabilities (GPU, storage type, CPU architecture).',
+);
+
+export const TARGET_NODE_SELECTOR_MODAL_DESCRIPTION = (
```

Comment:
If it's a component than it should be imported alone from the file, and should be named as the file with component naming convention like `TargetNodeSelectorModalDescription`

Reply:
I don't want to separate the details item help text and the modal help text just because one uses ForkliftTrans and the other don't, since once include the other, so I moved both into the `detailsItem` file.